### DISCUSSION TITLE
Add view to document type folder

### DISCUSSION THREAD

**Discussion by bjarnef** [umbraco/Umbraco-CMS#11310] [Features and ideas]
Currently document type folder level doesn't have a specific view. However this could be a useful feature.

For example to get an overview of:

- Document types which are element types
- Document types with templates
- Document types which vary by culture

Maybe event some bulk operations here to delete or move document types?

**Comment by skttl** [comment]
https://github.com/umbraco/Umbraco-CMS/issues/8867
